paasta_tools/cli/cmds/local_run.py

```diff
@@ -705,20 +734,9 @@ def assume_aws_role(
             file=sys.stderr,
         )
         sys.exit(1)
-    try:
-        with open("/nail/etc/runtimeenv") as runtimeenv_file:
-            aws_account = runtimeenv_file.read()
-            # Map runtimeenv in special cases to proper aws account name
-            if aws_account == "stage":
-                aws_account = "dev"
-            elif aws_account == "corp":
-                aws_account = "corpprod"
-    except FileNotFoundError:
-        print(
-            "Unable to determine environment for AWS account name. Using 'dev'",
-            file=sys.stderr,
-        )
-        aws_account = "dev"
+
+    aws_account = resolve_aws_account_from_runtimeenv(aws_account)
```

Comment:
should this be something like:
```suggestion
    aws_account = aws_account or resolve_aws_account_from_runtimeenv(aws_account)
```
then we can remove the check for `if aws_account is not None` from `resolve_aws_account_from_runtimeenv()`